### ISSUE TITLE
[BotW] Extended Memory/Draw Distance pack causing crashes when loading into Vah Ruta

### ISSUE THREAD

**Issue by ASleepyCat** [ActualMandM/cemu_graphic_packs#495]
Using the Extended Memory and Draw Distance graphics packs using the highest draw distance settings will cause Cemu to crash when loading into Vah Ruta.
```
[01:10:55] ------- Init Cemu 1.22.4 -------
[01:10:55] Init Wii U memory space (base: 0x172e0040000)
[01:10:55] mlc01 path: D:\Games\Cemu\mlc01\
[01:10:55] Cemuhook version: 0.5.7.3
[01:10:55] CPU: AMD Ryzen 5 2600 Six-Core Processor            
[01:10:55] RAM: 16336MB
[01:10:55] Used CPU extensions: SSSE3, SSE4.1, AVX2, AES-NI
[01:10:56] Loading U-King.rpx
[01:10:56] ------- Init Vulkan graphics backend -------
[01:10:56] Using GPU: GeForce RTX 2060
[01:10:56] Driver version (as stored in device info): 73424000
[01:10:56] VulkanLimits: UBAlignment 64 nonCoherentAtomSize 64
[01:10:56] Vulkan device memory info:
[01:10:56] Heap 0 - Size 5980MB Flags 0x00000001
[01:10:56] Heap 1 - Size 8168MB Flags 0x00000000
[01:10:56] Heap 2 - Size 246MB Flags 0x00000001
[01:10:56] Memory 0 - HeapIndex 1 Flags 0x00000000
[01:10:56] Memory 1 - HeapIndex 0 Flags 0x00000001
[01:10:56] Memory 2 - HeapIndex 1 Flags 0x00000006
[01:10:56] Memory 3 - HeapIndex 1 Flags 0x0000000e
[01:10:56] Memory 4 - HeapIndex 2 Flags 0x00000007
[01:10:56] VK_FORMAT_R4G4B4A4_UNORM_PACK16 missing features: COLOR_ATTACHMENT COLOR_ATTACHMENT_BLEND
[01:10:56] VK_FORMAT_R4G4_UNORM_PACK8 missing features: COLOR_ATTACHMENT COLOR_ATTACHMENT_BLEND
[01:10:56] VK_FORMAT_R5G5B5A1_UNORM_PACK16 missing features: COLOR_ATTACHMENT COLOR_ATTACHMENT_BLEND
[01:10:58] Loaded RPL module u-king (checksum 0x6267BFD0)
[01:10:58] ------- Loaded title -------
[01:10:58] TitleId: 00050000-101c9500
[01:10:58] TitleVersion: v208
[01:10:58] Update path: <mlc01>\usr\title\0005000e\101c9500\
[01:10:58] AOC path:    <mlc01>\usr\title\0005000c\101c9500\
[01:10:58] Save path:   <mlc01>\usr\save\00050000\101C9500\user\
[01:10:58] Shader cache file: shaderCache\transferable\00050000101c9500.bin
[01:10:58] gameprofile path: gameProfiles\00050000101c9500.ini
[01:10:58] RPX hash (updated): fb7911ad
[01:10:58] RPX hash (base): dcac9927
[01:10:58] Graphic pack: Using modified RAM mapping 10000000-a0000000 (+1310720KiB)
[01:10:58] Graphic pack: Using modified RAM mapping a0000000-e0000000 (+589824KiB)
[01:10:58] Loaded module 'u-king' with checksum 0x6267bfd0
[01:10:58] RPL link time: 141ms
[01:10:58] HLE scan time: 93ms
[01:10:58] ------- Active settings -------
[01:10:58] CPU-Mode: Multi-core recompiler (gameprofile)
[01:10:58] Load shared libraries: true (gameprofile)
[01:10:58] Use precompiled shaders: auto (gameprofile)
[01:10:58] Full sync at GX2DrawDone: true
[01:10:58] Async compile: true
[01:10:58] Console language: English
[01:10:58] Vulkan-Info: Allocated additional memory for textures from device-local memory
[01:10:59] ------- Activate graphic packs -------
[01:10:59] Applying patch group 'BotW_DrawDistance_V208' (Codecave: 01800000-0180003c)
[01:10:59] Activate graphic pack: The Legend of Zelda: Breath of the Wild/Mods/Draw Distance [Presets: Extreme (2x, requires Extended Memory pack!),Ultra (1.5x, requires Extended Memory pack!),Much higher(-3)]
[01:10:59] Applying patch group 'BotW_ExtendedMemory_V208'
[01:10:59] Activate graphic pack: The Legend of Zelda: Breath of the Wild/Mods/Extended Memory
[01:10:59] ------- Init Audio backend -------
[01:10:59] DirectSound: available
[01:10:59] XAudio 2.8: available
[01:10:59] ------- Run title -------
[01:11:05] Shader cache loaded with 9468 shaders. Commited mem 1572MB. Took 6813ms
[01:11:06] Recompiler initialized. CPU extensions: LZCNT MOVBE AVX 
[01:11:06] Custom timer mode: QPC
[01:11:06] IOSU_CRYPTO: No otp.bin found. Online mode cannot be used
[01:11:06] IOSU_CRYPTO: No Seeprom.bin found. Online mode cannot be used
[01:11:06] IOSU_ACT: using account default in first slot
[01:11:13] Vulkan-Info: Allocated additional memory for textures from device-local memory
[01:11:23] Vulkan-Info: Allocated additional memory for textures from device-local memory
[01:11:23] Vulkan-Info: Allocated additional memory for textures from device-local memory

Crashlog for Cemu 1.22.4
Date: 20-01-2021 15:11:25

-----------------------------------------
   Stack trace
-----------------------------------------
Exception 0xc0000005 at 0x17297f68a27
cemu.exe at 0x7ff7120d0000

RAX=000000000312e2fc RBX=0000000000000000 RCX=0000000000000000 RDX=000000000312e2fc
RSP=00000172a0ae6830 RBP=0000000000000000 RDI=0000000070dc78d8 RSI=0000000000000000
R8 =0000000000000000 R9 =00000000694575b8 R10=0000000068d678fc R11=0000000000000000
R12=000000000312e2fc R13=00000172e0040000 R14=000000006acf7cc4 R15=000001743f880000

-----------------------------------------
   Game info
-----------------------------------------
Game: Breath of the Wild [EU v208]
TitleId: 50000101c9500
RPXHash: dcac9927

-----------------------------------------
   Active PPC instance
-----------------------------------------
IP 0x0396bf48 LR 0x0312e2fc Thread 0x6acf8318

-----------------------------------------
   PPC stack trace
-----------------------------------------
[01:11:25] Trace starting at SP/r1 6acf7cc4
[01:11:25] SP 6acf7e34 ReturnAddr 037c18a8
[01:11:25] SP 6acf7f7c ReturnAddr 037c1cf4
[01:11:25] SP 6acf7fac ReturnAddr 02c43e70
[01:11:25] SP 6acf8174 ReturnAddr 02c4fcec
[01:11:25] SP 6acf818c ReturnAddr 02c4fa6c
[01:11:25] SP 6acf81ac ReturnAddr 02c30a50
[01:11:25] SP 6acf824c ReturnAddr 02c4f598
[01:11:25] SP 6acf8254 ReturnAddr 02c4f5f0
[01:11:25] SP 6acf825c ReturnAddr 037d8950
[01:11:25] SP 6acf826c ReturnAddr 037d8460
[01:11:25] SP 6acf827c ReturnAddr 037d75f8
[01:11:25] SP 6acf82b4 ReturnAddr 037d7efc
[01:11:25] SP 6acf82c4 ReturnAddr 030bcb6c
[01:11:25] SP 6acf82d4 ReturnAddr 030bbee0
[01:11:25] SP 6acf82e4 ReturnAddr 00e00000

-----------------------------------------
   PPC stack dump
-----------------------------------------
[0x6acf7cc4] 6acf7e34 0312e2fc 00010100 000027d7 - 68d678fc 10263910 102848d4 10263910
[0x6acf7ce4] 10284814 10263910 102848f0 10263910 - 10284878 10263910 10284920 10263910
[0x6acf7d04] 1028488c 10263910 0000000c 6acf7dec - 102848a0 10263910 6acf7d34 00000000
[0x6acf7d24] 10284994 10263910 00000000 7a8f7b24 - 9daacca2 00000673 458ef302 430fecb4
[0x6acf7d44] 43e15288 454ec000 4401e667 c4bb8333 - 454ec000 4401e667 c4bb8333 4566602a
[0x6acf7d64] 4405c000 c3767da2 6acf7d78 10284648 - 00000020 4d617054 6f776572 5f313100
[0x6acf7d84] 00b4820c 1031bc1c 10263910 649c431c - 00000000 7a820ef0 00000000 00000000
[0x6acf7da4] 00000000 00000000 00000000 10263950 - 1031bd0c 649c431c 00000002 10470000
[0x6acf7dc4] 00000000 10480000 10263910 00000000 - 70b23cb0 70b23ab8 1051365c 00000000
[0x6acf7de4] 00000000 00000000 00000000 10263910 - 00000000 00000000 00000000 00000000
[0x6acf7e04] 00000000 00000000 00000000 6acb45d8 - 00000000 00000000 00000000 00000000
[0x6acf7e24] 00000000 00000000 00000000 00000000 - 6acf7f7c 037c18a8 00000020 1031c2e4
[0x6acf7e44] 10263910 1031c2c4 10263910 03b4abe4 - 00000001 00000027 6acf7e00 6acf7e6c
[0x6acf7e64] 10284ab0 00000080 506c6163 656d656e - 742f4d61 696e4669 656c6444 756e6765
[0x6acf7e84] 6f6e2e62 1031c2d4 10263910 70b23ab8 - 70b23ab8 649c431c 00000d48 00001770
[0x6acf7ea4] 3e4ccccd 01cf7eb4 6acf7eb8 1031bd0c - 00000040 4d61696e 4669656c 6444756e

-----------------------------------------
   PPC threads
-----------------------------------------
0e060f50 Ent 00e00008 IP 00e00008 LR 00e00000 WAITING   Aff 111 Pri 32 Name Alarm Thread
0e004dc0 Ent 00e0000c IP 00e0000c LR 00e00000 WAITING   Aff 111 Pri 32 Name Callback Thread
0e081cec Ent 00e05b8c IP 00e00058 LR 031faa14 READY     Aff 010 Pri 80 Name Default Core 1
109f9178 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 010 Pri 84 Name Prepare Thread
15dbc6c0 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 80 Name WorkerMgr/Worker1(Any)
15dfcf98 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 80 Name WorkerMgr/Worker2(Any)
18e07e30 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 010 Pri 81 Name OverlayArena Prepare
18e09b18 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 83 Name DecompThread
5459dbb8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 83 Name Resource Loading
545a87c8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name Resource Control
545b3548 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name Resource Memory
545be438 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name MovableMemory
5ed25fc8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name res::Compaction
1059e0c0 Ent 042180cc IP 00e04554 LR 04218258 WAITING   Aff 100 Pri 67 Name nw::snd::TaskThread
0e036a68 Ent 00e05bc4 IP 00e05bc4 LR 00e00000 WAITING   Aff 100 Pri 14 Name NULL
66234200 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 82 Name aal::LowPrioWorkerThread
66623d20 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 72 Name AsyncPatternRumbleThread
670e8e38 Ent 030bbea8 IP 00e0468c LR 038ded84 WAITING   Aff 010 Pri 86 Name ProductReportBackGround Thread
67318398 Ent 030bbea8 IP 00e0468c LR 03429ed8 READY     Aff 010 Pri 83 Name NFP Thread
10e15530 Ent 03fa39f0 IP 00e046a8 LR 03f7b168 WAITING   Aff 001 Pri 80 Name Physics:HavokWorkerEntity(Core2)
10e16d60 Ent 03fa39f0 IP 00e046a8 LR 03f7b168 WAITING   Aff 100 Pri 80 Name Physics:HavokWorkerEntity(Core0)
1109fb30 Ent 03fa39f0 IP 00e046a8 LR 03f7b168 WAITING   Aff 001 Pri 80 Name Physics:HavokWorkerSensor(Core2)
1109f320 Ent 03fa39f0 IP 00e046a8 LR 03f7b168 WAITING   Aff 100 Pri 80 Name Physics:HavokWorkerSensor(Core0)
674280c8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name NavMeshSystemThread
674a4678 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 010 Pri 84 Name 
674cf728 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name 
674fa7d8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 84 Name 
67613500 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 82 Name SaveMgrThread
6aaf9150 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 81 Name WorkerSupport[0]
6ab1a460 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 81 Name WorkerSupport[1]
6ab1c000 Ent 030bbea8 IP 00e0468c LR 031ff784 WAITING   Aff 001 Pri 79 Name Sleeper
6ac294d8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 84 Name ActorCreate
6ac008c0 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name ActorCreate
6acf8318 Ent 030bbea8 IP 03420978 LR 0342093c RUNNING   Aff 010 Pri 81 Name GameScen TaskMgr
6f134c18 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 001 Pri 84 Name uiLowPrioThreadMgr
6abd7ca8 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 010 Pri 84 Name ActorCreate
6b6f2d08 Ent 030bbea8 IP 00e04554 LR 030decd0 WAITING   Aff 100 Pri 84 Name tera::ImageResourceMgr
```

**Comment by ASleepyCat** [comment]
Yeah they're up-to-date.

**Comment by ASleepyCat** [comment]
Fixed with fe82cd0?

**Comment by Crementif** [comment]
Oh, right. Yup, should be fixed now. Thanks for reporting the issue :)
